paddle/fluid/framework/executor.cc

```diff
@@ -68,7 +63,11 @@ ExecutorPrepareContext::~ExecutorPrepareContext() {
   VLOG(5) << "destroy ExecutorPrepareContext";
 }
 
-Executor::Executor(const platform::Place& place) : place_(place) {}
+Executor::Executor(const platform::Place& place) : place_(place) {
+  cur_step_ = 0;
+  open_ctx_cache_ = false;
+  ctx_is_cached_ = false;
```

Comment:
could we do a nullptr checking of ctx_ to judge whether ctx_is_cached is false? And we should pay attention to caching multiple ctx